### PR TITLE
update changelog for 5.0.0-pre.1

### PR DESCRIPTION
**Pull request by vkovec**

### CODE REVIEW

**Review by mfe — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,23 @@
 # Changes in Fbx Exporter
 
-## [Unreleased]
+## [5.0.0-pre.1]
```
Comment (mfe) [inline CHANGELOG.md:3]: @vkovec the entry is missing the date, so package validation is failing because of that.
Thanks

**Review by mfe — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mfe) [reply CHANGELOG.md]: I fixed it ✋

Comment (vkovec) [reply CHANGELOG.md]: thank you!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+
 ### Changed
 - Added support for exporting texture offset, scale and wrap mode. Thank you to @hybridherbst for the additions.
+- Updated the minimum Unity version to 2020.3.
+- Updated the minimum Recorder version to 4.0.0.
+- Updated the FBX SDK bindings to 5.0.0-pre.1.
+
+### Fixed
+- Cache result of SelectionContainsPrefabInstance in Editor Window to avoid unnecessary recalculations.
+- Use lists instead of dictionaries for property channel maps.
```
Comment (benoithudson) [inline CHANGELOG.md:18]: "in order to get consistent order of operations"

(not that we ever saw a problem but it was part of my worry; when you iterate over a hash table you iterate in an order that can vary between systems and even between runs of the program on the same system.)

File: com.unity.formats.fbx/Documentation~/recorder.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ With the FBX Exporter and the [Unity Recorder](https://docs.unity3d.com/Packages
 
 ## Requirements
 
-* To use the FBX Recorder, you must install the [Unity Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@latest/) package (version 2.5.5 or higher).
+* To use the FBX Recorder, you must install the [Unity Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@latest/) package (version 4.0.0 or higher).
```
Comment (benoithudson) [inline com.unity.formats.fbx/Documentation~/recorder.md:7]: This requires Unity 2021 right? I'd document that if it's true.

**Review by benoithudson — APPROVED** [review]
I'm happy with it, couple of minor wording fixes you can add if you feel like it.

**Pull request merged**